### DISCUSSION TITLE
Opening app shows Androids App-Info instead of opening the app

### DISCUSSION THREAD

**Discussion by paurf** [FossifyOrg/General-Discussion#418] [Q&A] [answered]
Hi, 
not sure if this is a bug, so I'll start a question first. 
I'm currently switching from SMT Calendar to Fossify Calendar. I have added a Fossify Calendar widget to my home screen and everything seems working fine; the app opens if I tap the widget. 
However, if I use the app symbol it doesn't open the app but rather the Andoid App-Info settings instead. Even after deleting the Widget, it still just opens the App Info, i.e. I can't access the calendar app at all (until creating a new widget). 
Reinstallation of the App helped, and now it appears to work with the App symbol as well again. 
Any idea what went wrong? Did I do something wrong?

**Comment by Aga-C** [comment]
What app version are you using? And have you changed the app's theme in Settings?

**Reply by paurf** [reply]
I have downloaded the latest (only?) version from F-Droid, i.e. version 1.0.2. I'm using a Pixel 4a with latest stock Android. 
I changed the theme to dark mode and edited the colors. This worked in SMT calender without problems and does work in Fossify calender after the reinstallation as well. I can't remember if I changed the app symbols color, but I think I didn't.

**Reply by Aga-C** [reply, marked as answer]
If you encounter this issue again, please raise a bug with a detailed description of what have you done. There was such a bug in a previous version, but it was fixed.

**Reply by paurf** [reply]
Thanks for your prompt help!
